tests/Sinch.Tests/Verification/VerificationTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.Json;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Newtonsoft.Json.Linq;
+using RichardSzalay.MockHttp;
+using Sinch.Verification.Start;
+using Sinch.Verification.Report;
+using Sinch.Verification.Report.Response;
+using Xunit;
+
+namespace Sinch.Tests.Verification
+{
+    public class VerificationTests : VerificationTestBase
+    {
+        private JsonSerializerOptions _jsonSerializerOptions;
+
+        [Fact]
+        public void DeserializeVerificationStartResponse()
+        {
+            var data = new
+            {
+                id = "1234567890",
+                method = "sms",
```

Comment:
Maybe you can use the `VerificationType` constants if it makes sense in the tests.

Reply:
That's anonymous object which usually represent raw data for my tests